src/main/java/uk/co/ramp/covid/simulation/place/Nursery.java

```diff
@@ -29,13 +30,7 @@ public void reportInfection(Time t, Person p, DailyStats s) {
     }
 
     @Override
-    public boolean isFullyStaffed() {
-        return nStaff > 0;
-    }
-
-    @Override
-    public Shifts getShifts() {
-        nStaff++;
-        return Shifts.schoolTimes();
+    protected List<BuildingTimeParameters> getTimeInfo() {
+        return PopulationParameters.get().buildingProperties.schoolTimes;
```

Comment:
I would argue that school opening times and nursery opening times may be different.